src/styles/more-light/theme.pcss

```diff
@@ -1466,13 +1475,7 @@
   height: 20px;
 }
 .p-radiobutton .p-radiobutton-box {
-  border: calc(var(--border-weight)*2) var(--border-style) var(--bluegray-100);
-  background: var(--surface-a);
-  width: 20px;
-  height: 20px;
-  color: var(--text-color);
-  border-radius: var(--border-radius--rounded);
-  transition: background-color 0.2s, color 0.2s, border-color 0.2s, box-shadow 0.2s;
+  display: none;
```

Comment:
Why remove all the declared styles and hide it?

Reply:
The version updated showed the File-Chooser from the browser, this hides it again. Why I cannot answer